### PR TITLE
Update and improve docs

### PR DESCRIPTION
**Pull request by ashcherbakov**
- Fix existing docs
  - dclcli -> dcld
  - .dcld and .dclcli -> .dcl
  - add missing commands (compliance provision, vendorinfo)
  - fixed command parameters
  - removed queries not implemented anymore
- Simplified and improved docs, get rid of redundant duplicates
- Fixes in vendorinfo
  - fixes REST endpoint (vendor/{vid} -> vendors/{vid})
  - unified state key prefix
- Other minor fixes

### CODE REVIEW

File: README-DEV.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ The easiest way to run a local pool is to start it in Docker:
 
     make install
 
-    # DCL_OBSERVERS=1 make localnet_init  # to initialize observers as well
```
Comment (andkononykhin) [inline README-DEV.md:73]: it was an optional thing

**Review by andkononykhin — COMMENTED** [review]
I believe few issues exist here, please check my comments

File: README-DEV.md (hunk 1 above)
Comment (ashcherbakov) [reply README-DEV.md:73]: Thank you, fixed

File: README-DEV.md [hunk 2]
```diff
@@ -83,16 +83,23 @@ This will start a local pool of 4 validator nodes in Docker. The nodes will expo
...
-dclcli
+dcld
 ```
-Have a look at [How To](docs/how-to.md) and [CLI Help](docs/cli-help.md) for instructions how to configure and use the CLI.
+Have a look at [How To](docs/how-to.md) and [transactions](docs/transactions.md) for instructions how to configure and use the CLI.
+
+## REST
+Start a local pool as described above.
+
+Every node exposes a REST API at `http://<node-ip>:26640`.
```
[the remark this answers is not in the document]
Comment (ashcherbakov) [reply README-DEV.md]: Thank you, fixed

File: docs/advanced/running-genesis-node.md [hunk 3]
```diff
@@ -27,29 +27,25 @@ The latest release can be found at [DCL Releases](https://github.com/zigbee-alli
...
-3. Configure dclcli:
-    * `dclcli config chain-id <chain-id>` - the chosen unique chain ID.
-    * `dclcli config output json` - Output format (text/json).
-    * `dclcli config indent true` - Add indent to JSON response.
-    * `dclcli config trust-node false` - Verify proofs for node responses.
-    * `dclcli config node tcp://localhost:26657` - Address of the genesis node (this is a local node). 
+3. Configure CLI:
+    * `dcld config chain-id <chain-id>` - the chosen unique chain ID.
+    * `dcld config output json` - Output format (text/json).
+    * `dcld config node tcp://localhost:26657` - Address of the genesis node (this is a local node).
```
[the remark this answers is not in the document]
Comment (ashcherbakov) [reply docs/advanced/running-genesis-node.md]: Done

File: docs/how-to.md [hunk 4]
```diff
@@ -43,26 +43,21 @@ the following instructions can be used for every role (see [Use Case Diagrams](u
...
+* `dcld config chain-id <chain-id>` - `<chain-id>` defines the Network you want to connect to
     * Use `testnet` if you want to connect to persistent Test Net
     * A full list of available persistent chains can be found in [Persistent Chains](../deployment/persistent_chains)
     where every sub-folder matches the corresponding chain-id.
-* `dclcli config output json` - Output format (text/json).
-* `dclcli config indent true` - Add indent to JSON response.
-* `dclcli config trust-node false` - Verify proofs for node responses.
-* `dclcli config node <address>` - Address of a node to connect.
+* `dcld config output json` - Output format (text/json).
+* `dcld config node <address>` - Address of a node to connect.
```
[the remark this answers is not in the document]
Comment (ashcherbakov) [reply docs/how-to.md]: Thank you, fixed

File: docs/transactions.md [hunk 5]
```diff
@@ -1430,20 +951,15 @@ If more than 1 Trustee signature is required to add the account, the account
...
-    - `address`: string // account address; bech32 encoded
-    - `pub_key`: string // account public key; bech32 encoded
-    - `vid`: 16 bit number // vendor id (only needed for vendor role)
-    - `roles`: array<string> // the list of roles to assign to account 
-- In State:
-  - `auth` store  
-  - `1:<address>` : `<account info> + <list of approvers>`
-  - `2:<address>` : `<account info>` (if just 1 Trustee is required)  
+    - address: `string` - account address; Bench32 encoded
+    - pub_key: `string` - account public key; Bench32 encoded
```
Comment (ankur325) [inline docs/transactions.md]:
```suggestion
    - pub_key: `string` - Account's Protobuf JSON encoded public key
```

File: docs/transactions.md [hunk 6]
```diff
@@ -1496,81 +1003,97 @@ Approves the proposed revocation of the account.
...
 
-Gets all proposed but not approved accounts.
+Gets a proposed but not approved accounts by its address
 
-- Parameters: No
+- Parameters:
+    - address: `string` - account address; Bench32 encoded
 - CLI command: 
-    -   `dclcli query auth all-proposed-accounts .... `
+    -   `dcld query auth proposed-account --addres <bench32 encoded string>`
```
Comment (ankur325) [inline docs/transactions.md]:
```suggestion
    -   `dcld query auth proposed-account --address <bench32 encoded string>`
```

**Review by ankur325 — COMMENTED** [review]
Few minor suggestions

File: docs/transactions.md [hunk 7]
```diff
@@ -11,1413 +11,934 @@ See use case sequence diagrams for the examples of how transaction can be used.
...
 
-Proposes a new self-signed root certificate.
+Adds a record about a Vendor.
 
-If it's sent by a non-Trustee account, or more than 1 Trustee signature is required to add a root certificate, 
-then the certificate
-will be in a pending state until sufficient number of other Trustee's approvals is received.
+- Parameters:
+  - vid: `uint16` -  Vendor ID (positive non-zero)
+  - vendorName: `optional(string)` -  Vendor name
```
[the remark this answers is not in the document]
Comment (ashcherbakov) [reply docs/transactions.md]: Thank you, fixed

File: docs/transactions.md [hunk 8]
```diff
@@ -11,1413 +11,934 @@ See use case sequence diagrams for the examples of how transaction can be used.
...
+  - companyLegalName: `optional(string)` -  Legal name of the vendor company
```
[the remark this answers is not in the document]
Comment (ashcherbakov) [reply docs/transactions.md]: Thank you, fixed

File: docs/transactions.md [hunk 9]
```diff
@@ -11,1413 +11,934 @@ See use case sequence diagrams for the examples of how transaction can be used.
...
+  - companyPreferredName: `optional(string)` -  Preferred name of the vendor company
+  - vendorLandingPageURL: `optional(string)` -  URL of the vendor's landing page
+- In State: `vendorinfo/VendorInfo/value/<vid>`
 - Who can send: 
-    - Any role
-- The current number of required approvals: 
-    - 2
+    - Vendor account who created initial entry
```
[the remark this answers is not in the document]
Comment (ashcherbakov) [reply docs/transactions.md]: Fixed

File: docs/transactions.md (hunk 6 above)
Comment (ashcherbakov) [reply docs/transactions.md]: Thank you, fixed

File: docs/transactions.md [hunk 10]
```diff
@@ -1496,81 +1003,97 @@ Approves the proposed revocation of the account.
...
+
+#### GET_PROPOSED_ACCOUNT_TO_REVOKE
+**Status: Implemented**
+
+Gets a proposed but not approved accounts to be revoked by its address.
+
+- Parameters:
+    - address: `string` - account address; Bench32 encoded
+- CLI command: 
+    -   `dcld query auth proposed-account-to-revoke --addres <bench32 encoded string>`
```
[the remark this answers is not in the document]
Comment (ashcherbakov) [reply docs/transactions.md]: Thank you, fixed

File: docs/transactions.md (hunk 5 above)
Comment (ashcherbakov) [reply docs/transactions.md:955]: I think account's public key is a bench32 encoded key, so it seems a correct description

Comment (ankur325) [reply docs/transactions.md:955]: While testing found that it needed full JSON encoded public key.

Comment (ashcherbakov) [reply docs/transactions.md]: Yes, you are right, thank you. Fixed.

File: docs/transactions.md [hunk 11]
```diff
@@ -1430,20 +951,15 @@ If more than 1 Trustee signature is required to add the account, the account
...
+    - vid: `optional(uint16)` - vendor id (only needed for vendor role)
+    - roles: `array<string>` - the list of roles, comma-separated, assigning to the account. Supported roles: `Vendor`, `TestHouse`, `CertificationCenter`, `Trustee`, `NodeAdmin`. 
+- In State: `dclauth/PendingAccount/value/<address>`
 - Who can send: 
     - Trustee
 - CLI command: 
-    -   `dclcli tx auth propose-add-account --address=<account address> --pubkey=<account pubkey> --roles=<role1,role2,...> --vid=<uint16> --from=<trustee name>`
-- REST API: 
-    -   POST `/auth/accounts/proposed`
+    -   `dcld tx auth propose-add-account --address=<bench32 encoded string> --pubkey=<bench32 encoded string> --roles=<role1,role2,...> --vid=<uint16> --from=<account>`
```
[the remark this answers is not in the document]
Comment (ashcherbakov) [reply docs/transactions.md]: Fixed

**Pull request merged**